python/ray/air/_internal/torch_utils.py

```diff
@@ -1,71 +1,31 @@
-import os
 import warnings
 from typing import Dict, List, Optional, Union, Any
 
 import numpy as np
 import pandas as pd
-import torch
 
-import ray
+from ray.air._internal.accelerator_utils import try_import_torch
 from ray.air.util.data_batch_conversion import _unwrap_ndarray_object_type_if_needed
 
-from ray._private.accelerators.hpu import HPU_PACKAGE_AVAILABLE
 
-if HPU_PACKAGE_AVAILABLE:
-    import habana_frameworks.torch.hpu as torch_hpu
+torch, _ = try_import_torch()
```

Comment:
Can you help me understand the difference between `torch` and `torch_npu`/`torch_hpu`? 

Can we still `import torch` first, then import `torch_npu`/`torch_hpu` as additional libraries? Then probably having something like `import_accelerator_torch_library` to decouple this two types of import would be better.

Reply:
First, please see the previous [comment](https://github.com/ray-project/ray/pull/44086#discussion_r1545940380) for the difference between `torch` and `torch_npu`.

Good suggestion, we use `try_import_accelerator_torch_library` to decouple these.